docs/config/auth0.md

```diff
@@ -0,0 +1,74 @@
+# Auth0 setup
+
+This is a documentation of the Auth0 and identity provider configuration that the ASAP hub uses.
+The aim is to make the configuration reproducible based on the content in here, instead of hidden in someone's Auth0 account.
+This document is targeted at developers and does _not_ aim to explain OAuth 2, OIDC, or Auth0. For a guide about those, please refer to e.g. Auth0 documentation.
+
+## Auth0 frontend application
+
+An Auth0 application for the Hub frontend needs to be created
+and configured as follows:
+
+**Settings**
+
+- `domain` - pick any and copy to the frontend auth config
+- `Client ID` - generated, copy to the frontend auth config
+- `Application Type` - Single Page Application
+- `Allowed Callback URLs` - `http://localhost:3000, https://$PRODUCTION_DOMAIN_NAME`
+- `Allowed Web Origins` - `http://localhost:3000, https://$PRODUCTION_DOMAIN_NAME`
+
+**Connections**
+
+- `Username-Password-Authentication` - `true`
+- `google-oauth2` - `true`
+
+## Auth0 Universal Login
+
+**Settings**
+
+- `Company Logo` - URL to the ASAP logo
+- `Primary Color` - pick any matching branding
+- `Background Color` - pick any matching branding
+
+**Login**
+
+- `Customize Login Page` - `true`
+- `HTML` - [here](auth0/login.html)
```

Comment:
Is this the HTML you mentioned we could generate with the same components used on the website?

Reply:
Yes, this is basically the Auth0 default HTML with some slight changes made for ORCID